web/tests/markdown.test.cjs

```diff
@@ -412,13 +412,14 @@ test("marked", ({override}) => {
                 '<p>This is a <a class="stream-topic" data-stream-id="1" href="/#narrow/channel/1-Denmark/topic/some.20topic">#Denmark &gt; some topic</a> stream_topic link</p>',
         },
         {
-            input: "This has two links: #**Denmark>some topic** and #**social>other topic**.",
+            input: "This is a #**Denmark>** stream_topic link with empty string topic.",
             expected:
-                '<p>This has two links: <a class="stream-topic" data-stream-id="1" href="/#narrow/channel/1-Denmark/topic/some.20topic">#Denmark &gt; some topic</a> and <a class="stream-topic" data-stream-id="2" href="/#narrow/channel/2-social/topic/other.20topic">#social &gt; other topic</a>.</p>',
+                '<p>This is a <a class="stream-topic" data-stream-id="1" href="/#narrow/channel/1-Denmark/topic/">#Denmark &gt; </a> stream_topic link with empty string topic.</p>',
```

Comment:
This needs an API design discussion; we may want to have a special text label for this other than `Denmark > `.